css/index.scss

```diff
@@ -2,69 +2,126 @@ body {
     margin: 0;
     padding: 0;
     height: 100vh;
+    width: 100vw;
     font-size: 16px;
-    background-color: #efefef;
+    background-color: #3c40c6;
+    color: white;
+    font-family: Helvetica, sans-serif;
 }
 
-.container {
-    display: grid;
-    grid-template-columns: repeat(3, 1fr);
-    grid-template-rows: repeat(3, 33.3vh);
-
-    .username {
-        font-size: 2em;
-        grid-column: 2;
-        grid-row: 2;
-        justify-self: center;
-
-        h1 {
-            margin: 0;
-        }
-
-        input {
-            width: 50%;
-        }
-    }
-
-    .settings {
-        grid-column: 1;
-        grid-row: 1;
-        padding: 1em;
-    }
-
-    .weather {
-        grid-column: 3;
-        grid-row: 1;
-        padding: 1em;
-        justify-self: right;
-    }
-
-    .todo-list {
-        grid-column: 1;
-        grid-row: 2/4;
-        align-self: end;
-        padding: 1em;
-
-        .todo-list-card {
-            background-color: #fff;
-            width: auto;
-            padding: 1px 0 0 40px;
-            border-radius: 2em;
-            box-shadow: #d6d6d6 3px 5px;
-    
-            ul {
-                margin: 0;
-                padding: 0;
-    
-                li {
-                    list-style: none;
-                    padding: 12px 8px 12px 40px;
-                    font-size: 18px;
-                    user-select: none;
-                }
-            }
-        }
-
-        
-    }
+.container{
+    display: flex;
+    width: 100%;
+    height: 100vh;
+    flex-direction: row;
+    justify-content: space-between;
+
+}
+
+.left{
+    display: flex;
+    flex-direction: column-reverse;
+    margin: 15px;
+}
+
+.left, .mid, .right{
+  width:33%;
+}
+
+.todo-list{
```

Comment:
Nest everything that has with todo to do together (lol this felt so weird to write)